model_templates/training/r_lang/README.md

```diff
@@ -0,0 +1,14 @@
+## R Fit Template
```

Comment:
As long as you already have this model template, could you add a functional test case?
https://github.com/datarobot/datarobot-user-models/blob/jlpolit/r_fit_draft/tests/functional/test_training_model_templates.py

Reply:
I may be wrong here, but I thought we needed to publish the new version of DRUM and update that import in the R drop-in env for the functional test case to work?

Reply:
Yes, you are right.
We can do the following: I take this branch,
1. Compile and publish drum 1.1.4rc1,
2. Update all the envs in current branch to use new drum and push into this PR. Functional tests then will use this new drum.
3. Once everything passes, we merge PR and install new envs on staging.

Just tell me when it is ready, so i can take it and publish drum.